media-fonts/fonts-meta/metadata.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>juippis@gentoo.org</email>
+    <name>Fonts</name>
+  </maintainer>
+  <use>
+    <flag name="free">Install fonts for the "free" preset</flag>
```

Comment:
What are these "free" and "ms" presets?

Reply:
No clue, probably some old language not so relevant anymore.
https://hub.mpcthemes.net/knowledgebase/typography-preset/

What do you suggest as the replacement?

Reply:
I'm not sure how that website's presets are applicable here...

If I knew what these meant, I'd provide some suggestion :)